docs/configuring-metabase/localization.md

```diff
@@ -6,6 +6,54 @@ redirect_from:
 
 # Languages and localization
 
+The **Localization** settings allow you to set global defaults for your Metabase instance. You can find **Localization** under **Admin settings** > **Settings**.
+
+## Instance language
+
+The instance language is the default language across the Metabase UI, system [emails](./email.md), [dashboard subscriptions](../dashboards/subscriptions.md), and [alerts](../questions/sharing/alerts.md). People can pick a different language from their own [account settings](../people-and-groups/account-settings.md).
```

Comment:
```suggestion
Here you can set the default language across the Metabase UI, system [emails](./email.md), [dashboard subscriptions](../dashboards/subscriptions.md), and [alerts](../questions/sharing/alerts.md). People can pick a different language from their own [account settings](../people-and-groups/account-settings.md).
```

Additionally, we should follow this section with the list of supported languages, as the natural follow-up to setting a default language is 'which languages are available'?